File: 1502.can-make-arithmetic-progression-from-sequence/solution.py
class Solution:
    def canMakeArithmeticProgression(self, arr: List[int]) -> bool:
        arr.sort()
        diff = arr[1] - arr[0]
        N = len(arr)
        for i in range(1, N - 1):
            if arr[i] + diff != arr[i + 1]:
                return False
        return True

File: 1502.can-make-arithmetic-progression-from-sequence/solution.kt
class Solution {
    fun canMakeArithmeticProgression(arr: IntArray): Boolean {
        arr.sort()
        val diff = arr[1] - arr[0]
        for (i in 1 until arr.size - 1) {
            if (arr[i] + diff != arr[i + 1]) return false
        }
        return true
    }
}
